parity/reports/algod2indexer_all_human.txt

```diff
@@ -0,0 +1,210 @@
+INDEXER--->definitions.Account.properties.closed-at-round:{"description":"Round during which this account was most recently closed.","type":"integer","x-algorand-format":"uint64"}
```

Comment:
This report shows all the "essential" difference between **algod** and **indexer** in a purportedly human-friendly format. There are variants of this report that limit the scope to:
* changes of common attributes
* attributes which are not present in **algod** but exist in **indexer**
* attributes which are present in **algod** but missing from **indexer**